model/riscv_vlen.sail

```diff
@@ -21,9 +25,13 @@ function get_elen_pow() = match elen {
 
 register vlen : bits(4)
 
+val sys_vector_vlen_exp = pure "sys_vector_vlen_exp" : unit -> bits(4)
```

Comment:
There's no reason to use `bits(4)` here. This can simply return `range(3, 16)` (I'm pretty sure the minimum is 3 not 5?).

I think the elen exponent should also be `range(3, 16)`.